Richardjm/adxl chain

richardjm opened the pull request:
<!--
Thank you for your interest in contributing to the VoronUsers repository, it is
highly appreciated!

**Please make sure the submission conforms to the rules outlined below. PRs which fail
to conform to the rules below are likely to be rejected.**
-->

  * [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

<!--
Describe the submission further using the template provided below. The more 
details the better!
-->

#### Which mods/configurations/profiles are added by this PR?

A few variants of an ADXL chain mount for Adafruit/GY-291 for 2 or 3 whole chain fittings

#### How was it tested? 

Tested by myself and various other members of discord

#### Any background context you want to provide?

Started off inspired by https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/padok/chain_anchor-ADXL345_mount as I needed a generic chain version and snowballed from there.

#### Screenshots (if appropriate)

#### Further notes

whistlinric on discord

KiloQubit reviewed (changes requested):
Thanks for your mod!

In `ChainAnchor-Generic-ADXL345-Adafruit.stl` and `ChainAnchor-IGUS-ADXL345-Adafruit.stl` can you make this overhang less steep?

<img width="722" alt="Screen Shot 2021-10-02 at 16 38 13" src="https://user-images.githubusercontent.com/69389593/135731165-ea79f9ea-cdc4-4fe6-9ab8-0dfd10f461e2.png">

richardjm commented:
> 
> 
> Thanks for your mod!
> 
> In `ChainAnchor-Generic-ADXL345-Adafruit.stl` and `ChainAnchor-IGUS-ADXL345-Adafruit.stl` can you make this overhang less steep?
> 
> <img alt="Screen Shot 2021-10-02 at 16 38 13" width="722" src="https://user-images.githubusercontent.com/69389593/135731165-ea79f9ea-cdc4-4fe6-9ab8-0dfd10f461e2.png">

Fixed up, and realised the Adafruit board was a 0.1in hole anyway which is M2.5 - so moved to threading into plastic and updated readme.
![image](https://user-images.githubusercontent.com/2081919/135740927-aefa18c0-8967-4d15-a7c2-85cdcee6dca6.png)

KiloQubit reviewed (approved):
Thanks for the update!
